.env.j2

```diff
@@ -3,10 +3,4 @@ FLASK_APP=todo_app/app
 FLASK_ENV=production
 
 # Change the following values for local development.
-SECRET_KEY=secret-key
-
-# Trello api
-API_KEY={{ api_key }}
-API_TOKEN={{ api_token }}
-ORGANISATION_ID={{ organisation_id }}
-BOARD_ID={{ board_id }}
\ No newline at end of file
+SECRET_KEY=secret-key
```

Comment:
For future I'd not worry too much about keeping your Ansible up to date as we won't be using it again, but if we are updating this we should arguably add some references to the DB 😛